common/src/main/resources/assets/figura/lang/en_us.json

```
@@ -440,7 +441,7 @@
     "figura.config.text_scroll_delay": "Text Scroll Delay",
     "figura.config.text_scroll_delay.tooltip": "Changes the delay (in ticks) that the Text should wait before scrolling",
     "figura.config.reduced_motion": "Reduced Motion",
-    "figura.config.reduced_motion.tooltip": "Disables all screen animations, making them instant, or a single frame\nAlso makes tooltips show in the bottom center of the screen, instead of following the mouse\nDoes not affect animations that have their own setting (like background animation and text scroll)",
+    "figura.config.reduced_motion.tooltip": "Disables all screen animations, making them instant, or a single frame\nAlso makes tooltips show in the bottom center of the screen, instead of following the mouse\nDoes not effect animations that have their own setting (like background animation and text scroll)",
```

"affect" is correct here as it is being used as a verb.
```suggestion
    "figura.config.reduced_motion.tooltip": "Disables all screen animations, making them instant, or a single frame\nAlso makes tooltips show in the bottom center of the screen, instead of following the mouse\nDoes not affect animations that have their own setting (like background animation and text scroll)",
```